app/views/bulk_add_batches/new.html.erb

```diff
@@ -11,7 +11,7 @@
 <%= form_tag site_bulk_add_batches_path(@site), role: 'form', 'data-module' => 'toggle-mapping-form-fields' do %>
 
   <% if @batch.errors %>
-    <%= render 'shared/error_messages', error_messages: @batch.errors.values.flatten(1) %>
+    <%= render 'shared/error_messages', error_messages: @batch.errors.map { |error| error.message }.flatten(1) %>
```

Comment:
not sure what the preferred syntax is here, but could use `map(&:message)` here to cut off some characters, like I said, don't actually mind which is used, i just like the &: syntax :D

Reply:
Good idea, I've updated the relevant commit.